database_schema.md

```diff
@@ -75,6 +75,29 @@ person_id: {
 }
 ```
 
+### Body
+```
+body_id: {
+    name: str
+    description: str
+    created: datetime
+    ended: datetime
+    chair_person_id: str
+    external_source_id: str
+    is_select: bool
```

Comment:
Is `is_select` meant to capture whether the body is something like 'Select Committee on Campaign Reform'? That's fine if that is the case.

Reply:
Yeah it was supposed to be a flag for whether a committee was "Select" vs. "Standing" like [here](https://www.seattle.gov/council/committees). But could generalize this to `tag: str` for versatility in filtering if it's ever needed.

Reply:
Let's go with `tag`